File: WorldFeed/Server/src/WorldFeed.Common/Public/Events/UserWitheldEventArgs.cs
namespace WorldFeed.Common.Public.Events
{
    using System;

    using WorldFeed.Common.Public.Streaming.Events;

    /// <summary>
    /// Event informing that some user event happened by were blocked by the country
    /// </summary>
    public class UserWitheldEventArgs : EventArgs
    {
        public UserWitheldEventArgs(IUserWitheldInfo userWitheldInfo)
        {
            UserWitheldInfo = userWitheldInfo;
        }

        public IUserWitheldInfo UserWitheldInfo { get; }
    }
}

File: Sharebook/Client/src/app/core/Public/Events/UserWitheldEventArgs.ts
import {IUserWitheldInfo} from "../Streaming/Events/IUserWitheldInfo";

// Event informing that some user event happened by were blocked by the country
export class UserWitheldEventArgs /*extends EventArgs*/ {
  constructor(userWitheldInfo: IUserWitheldInfo) {
    // super();

    this.userWitheldInfo = userWitheldInfo;
  }

  public userWitheldInfo: IUserWitheldInfo;
}
